requirements.txt

```diff
@@ -1,54 +1,10 @@
-args==0.1.0
```

Comment:
Ah looks like we had a lot of stuff in here from when we briefly added some other deps that needed cleanup. Thanks for doing that!